broker/core/inc/com/centreon/broker/io/stream.hh

```diff
@@ -54,6 +54,8 @@ namespace io {
 class stream {
   const std::string _name;
 
+  bool _bbdo_encoding = true;
+
```

Comment:
The name is badly chosen, something like _grpc_serialized would look better I think.

Do we really need this variable to be in all streams?
If I understand well, you just use it in the grpc stream. Why not move it to the grpc stream and check if it is true or false in the grpc::factory?

Reply:
it's also used in compression and bbdo stream